src/Components/Navbar/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {Container} from './index.styles'
+class Navbar extends React.Component{
```

Comment:
new line

Reply:
defined class after giving some lines of spaces